app/presenters/tag_set_presenter.rb

```diff
@@ -166,8 +166,9 @@ def build_list_item(tag, current_query: "", highlight: false)
     html
   end
 
-  def tag_link(tag, link_text = tag.name)
+  def tag_link(tag, link_text = tag.name, link_type = :tag)
+    link_base = link_type == :wiki_page ? "/wiki_pages/show_or_new?title=" : "/posts?tags="
```

Comment:
Do `Rails.application.routes.url_helpers` up top and then you should be able to use the helpers like usual.